apps/server/src/shared/repo/news/news.repo.integration.spec.ts

```diff
@@ -301,4 +304,36 @@ describe('NewsRepo', () => {
 			await expect(repo.findOneById(failNewsId)).rejects.toThrow(NotFoundError);
 		});
 	});
+
+	describe('findByCreatorId', () => {
+		const setup = async () => {
+			const user = userFactory.buildWithId();
+			const news = teamNewsFactory.build({
+				creator: user,
+			});
+			const news2 = teamNewsFactory.build({
+				updater: user,
+			});
+
+			await em.persistAndFlush([news, news2]);
+			em.clear();
+
+			return { news, news2, user };
+		};
+		it('should find a news entity by creatorId', async () => {
```

Comment:
have a separate test for creator and one for updater

Reply:
i added one test case